### ISSUE TITLE
I can't connect to 1.19 servers because the client only exists in 1.19.2

### ISSUE THREAD

**Issue by MakrelenBoss** [jrbudda/Vivecraft_119#7]
Could you maybe release a 1.19 version, too?

**Comment by nalathethird** [comment]
Agreed, I use 1.19 for about every server and world I'm in, I would need a 1.19 version

**Comment by lumithespoofy64** [comment]
If you want to use specific versions, go here!

https://www.curseforge.com/minecraft/mc-mods/vivecraft

It also shows fabric support!
